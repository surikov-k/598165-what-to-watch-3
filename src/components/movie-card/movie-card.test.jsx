import React from 'react';
import renderer from 'react-test-renderer';

import MovieCard from './movie-card.jsx';

const movie = {
  id: 0,
  title: ``,
  genre: ``,
  release: 0,
  rating: 0,
  description: [``],
  director: [``],
  cast: [``],
  preview: ``,
  poster: ``,
  background: ``,
};

describe(`<MovieCard/>`, () => {
  it(`renders correctly`, () => {
    const component = renderer.
      create(<MovieCard
        movie={movie}
        onMouseEnter={() => {}}
        onClick={() => {}}
      />).
      toJSON();

    expect(component).toMatchSnapshot();
  });
});
