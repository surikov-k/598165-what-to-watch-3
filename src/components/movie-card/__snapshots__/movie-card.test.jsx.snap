// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieCard/> renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
>
  <div
    className="small-movie-card__image"
    onClick={[Function]}
  >
    <img
      alt=""
      height="175"
      src=""
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
    onClick={[Function]}
  >
    <a
      className="small-movie-card__link"
      href=""
    >
      
    </a>
  </h3>
</article>
`;
